README.md

```diff
@@ -5,9 +5,9 @@ Small repo done in cab ride
 A simple repo used to track progress of codechef monthly contests from JAN15-JUNE17 (Can be increased or decresed by modifying code)
 
 ## What to do?
-* Install virtual env
+* Install virtual env 
 * Create an environment using virtual env `virtualenv envName -p python3`
-* Activate the environment and install the requirements `source bin/activate; pip3 install -r req.txt`
+* Activate the environment and install the requirements `source envName/bin/activate; pip3 install -r req.txt`
```

Comment:
The only command one will ever run will be manage.py, so its better for user to be in the root dir of the django project. 
Keep helper.py at the same place as manage.py which would be envName/ folder. I feel its better to keep all runnables there. Then again, this is my first Django project, correct me if I'm wrong.